.gitignore

```diff
@@ -0,0 +1,71 @@
+versions.json
```

Comment:
Can you please pull in the changes from master into your branch, so that the diff here only shows the assets adde/modified by you in this PR? Currently they are showing the changes made by Akshay in his PR.

Reply:
Updated with base branch